core/repository/api/src/test/java/org/eclipse/rdf4j/repository/util/RDFLoaderTest.java

```diff
@@ -50,14 +50,14 @@
 
 /**
  * Unit tests for {@link RDFLoader}.
- * 
+ *
  * @author Manuel Fiorelli
  *
  */
 public class RDFLoaderTest {
 
 	@ClassRule
-	public static WireMockRule wireMockRule = new WireMockRule(wireMockConfig().dynamicPort().dynamicHttpsPort());
+	public static WireMockRule wireMockRule = new WireMockRule(wireMockConfig().dynamicPort());
```

Comment:
Moved from https to http because WireMock doesn't support https on aarch64 MacOS. 

https://issueexplorer.com/issue/google/conscrypt/1034